bundle/language_en.properties

```diff
@@ -1693,6 +1694,7 @@ angal.therapy.worksheet.btn
 angal.therapy.worksheet.btn.key                                                                        = W
 angal.userbrowser.addnewuser.title                                                                     = Add New User
 angal.userbrowser.deleteuser.fmt.msg                                                                   = Delete user: {0}?
+angal.userbrowser.softdeleteuser.fmt.msg                                                               = Could not delete the user '{0}' because he is used somewhere. Do you want to mark him as deleted ?
```

Comment:
```suggestion
angal.userbrowser.softdeleteuser.fmt.msg                                                               = Could not delete the user '{0}' because it is used somewhere. Do you want to mark him as deleted ?
```